fpga/include/villas/fpga/card.hpp

```diff
@@ -25,7 +25,6 @@ class Card {
 
   std::string name; // The name of the FPGA card
   std::shared_ptr<kernel::vfio::Container> vfioContainer;
-  std::shared_ptr<kernel::vfio::Device> vfioDevice;
```

Comment:
Could you please add a description to the PR? Why is the vfioDevice handle something speciic to PCIe cards? In my understanding VFIO is supposed to abstract beyond PCIe and also applies to Platform devices.